lib/dialects/mysql/query.js

```diff
@@ -110,6 +110,8 @@ Query.prototype.formatResults = function(data) {
     result = data[0].version;
   } else if (this.isForeignKeysQuery()) {
     result = data;
+  } else if (this.isInsertQuery() || this.isUpdateQuery()) {
+    result = [result, data.affectedRows];
```

Comment:
@mickhansen Yup, you're right, but I thought it was cleaner to maintain the same interface for both queries as it currently does.